rust/main/chains/hyperlane-sealevel/src/log_meta_composer/tests.rs

```diff
@@ -49,6 +69,27 @@ pub fn test_search_delivered_message_transaction() {
     assert!(!transaction_hashes.is_empty());
 }
 
+#[test]
+pub fn test_search_delivered_message_reverted_transaction() {
```

Comment:
nice, would be good to also test the multi-tx per block case, where 1 was successful and 1+ failed

Reply:
added